src/mlpack/core/data/scaler_methods/maxabsscaler.hpp

```diff
@@ -0,0 +1,108 @@
+/**
+ * @file maxabsscaler.hpp
+ * @author Jeffin Sam
+ *
+ * MaxAbsScaler class to scale features.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_CORE_DATA_MAX_ABS_SCALE_HPP
+#define MLPACK_CORE_DATA_MAX_ABS_SCALE_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack {
+namespace data {
+/**
+ * A simple MaxAbs Scaler class.
+ *
+ * Given an input dataset this class helps you to scale each
+ * feature by its maximum absolute value.
+ *
+ * @code
+ * arma::Mat<double> input = loadData();
+ * arma::Mat<double> output;
+ *
+ * // Scale the features using
```

Comment:
Hm, this doesn't really follow the style guidelines, since this isn't a complete sentence. I guess in this case we can remove "using" at end with a point.

Reply:
Sorry , I meant it using the function below , but made changes